service.go

```diff
@@ -61,7 +69,7 @@ func (s *Service) Run(ctx context.Context) error {
 
 func writeErrorReply(w io.Writer, err error) error {
 	responseNotOk := birch.EC.Int32("ok", 0)
```

Comment:
seems like this should be -1

Reply:
The mongo shell expects ok to be 0 if it's not ok and 1 if it is ok to let them write `if !ok` in JavaScript.